test/lib/aws-mock.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const AWSMock = require('aws-sdk-mock');
+
+module.exports = exports = {};
+
+exports.uploadMock = {
+  ETag: '"ddad2cbff1bd8023ed7cd42e5059600a"',
+  Location: 'https://artc-staging-assets.s3.amazonaws.com/03a7990e1adf9fa8944df58dfc81cc5b.jpg',
+  key: '03a7990e1adf9fa8944df58dfc81cc5b.jpg',
+  Key: '03a7990e1adf9fa8944df58dfc81cc5b.jpg',
+  Bucket: 'artc-staging-assets',
+};
+
+AWSMock.mock('S3', 'upload', function(params, callback){
```

Comment:
great work mocking s3 ... 
this enables your test to pass w/o depending on third party services to run :) ... it also save you money!